lang/ja.json

```diff
@@ -457,7 +457,7 @@
           "value": "{NUMBER}"
         },
         "bp": {
-          "label": "BP (Badge Points)",
+          "label": "BP(バッジポイント)",
```

Comment:
Probably should be either a space before the ( or it should be full width, what do you think?

Reply:
There are no uniform rules for full-width or half-width parentheses in Japanese text.
If there is, it is only necessary to unify full-width and half-width brackets.
If you use half-width brackets, you need to put a space before and after the brackets appropriately.